### PR TITLE
added delete cascade to deletePost

### PR DESCRIPTION
**Pull request by mnmin**

### CODE REVIEW

File: src/controllers/post.js [hunk 1]
```diff
@@ -152,13 +152,20 @@ export const deletePost = async (req, res) => {
...
+    })
+
     const deletePost = await dbClient.post.delete({
       where: {
         id
       }
     })
 
-    return sendDataResponse(res, 201, deletePost)
+    return sendDataResponse(res, 201, deleteComments, deletePost)
```
Comment (Auenc) [inline src/controllers/post.js:168]: What are you trying to do on this line? You're passing 4 arguments until a function that only accepts and uses 3 parameters.

Comment (mnmin) [reply src/controllers/post.js:168]: Hi, I'm trying to first delete the comments within a post and then to delete the post itself using cascading deletes from Prisma CRUD

Comment (Auenc) [reply src/controllers/post.js:168]: Okay. I'm just not sure why you are sending both `deleteComments` and `deletePost` to `sendDataResponse`. `sendDataResponse` will send the `deleteComments` variable to the user, but not the `deletePost`. Do we need to send the `deletePost` variable as well?

Comment (mnmin) [reply src/controllers/post.js:168]: Ok, I've understood:  ````return sendDataResponse(res, 201, { deletedPost, deletedComments })````
```

**Review by Noemi890 — APPROVED** [review]

**Pull request merged**